src/components/Cart/Cart.js

```diff
@@ -0,0 +1,198 @@
+                    <input
+                      type="radio"
+                      className="greenRadio"
+                      id="white"
+                      name="radio-button"
+                    />
+                    <label htmlFor="green">Green</label>
+                  </div>
+                  <div className="quantity">
+                    <PlusCircleOutlined
+                      className="quant-icon"
+                      onClick={() => {
+                        dispatch(inccQuantity(plant));
+                      }}
+                    />
+                    <h2>{plant.qty}</h2>
+                    <MinusCircleOutlined
+                      className="quant-icon"
+                      onClick={() => {
+                        dispatch(descQuantity(plant));
+                      }}
+                    />
+                  </div>
+                  <div className="rate">
+                    <h2>Rs {plant.price * plant.qty}</h2>
+                    <button
+                      onClick={() => {
+                        deleteItem();
+                        dispatch(removeFromCart(plant));
+                      }}
+                    >
+                      <DeleteOutlined /> Delete
+                    </button>
+                  </div>
+                </div>
+              ))}
+
+              <div className="totalItem">
+                <div className="endLine"></div>
+                <h2>Total {cart.length} Items</h2>
+              </div>
+            </div>
+          </div>
+
+          <div className="summary">
+            <h1>Summary</h1>
+            <div>
+              <input type="text" placeholder="Promocode" />
+              <button className="apply">Apply</button>
+            </div>
+            <div>
+              <input type="radio" name="promo" id="first" />
+              <label htmlFor="first">LUCKYUSER</label>
+            </div>
+            <div>
+              <input type="radio" name="promo" id="first" />
+              <label htmlFor="first">FIRST50</label>
+            </div>
+            <div className="line"></div>
+            <div className="total">
+              <h3>
+                Subtotal<span>Rs {total}</span>
+              </h3>
+              <h3>
+                Discount
+                <span className="discount">
+                  {cart.length === 0 ? "Rs 0" : "Rs 10(-)"}
+                </span>
+              </h3>
+              <h3>
+                Delivery<span> Rs {cart.length === 0 ? "0" : "40"}</span>
+              </h3>
+
+              <div className="line"></div>
+              <h2>
+                Total
+                <span>
+                  Rs {cart.length === 0 ? "0" : grandTotal.total + 30}
+                </span>
+              </h2>
+            </div>
+            {cart.length === 0 ? (
+              <button className="shopping">Continue Shopping</button>
+            ) : (
```

Comment:
ternary operatoor , and operator. use component

Reply:
Done